roles/sap_ha_install_anydb_ibmdb2/tasks/passwordless_ssh.yml

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
There is a collection 

https://galaxy.ansible.com/ui/repo/published/community/crypto

Looks like it is quite reliable collection to handle ssh keys and other encryption related tasks. Why not use that collection instead of commands.